tests/providers/slack/hooks/test_slack.py

```diff
@@ -120,7 +120,7 @@ def test_call_with_failure(self, slack_client_class_mock):
         test_api_params = {'key1': 'value1', 'key2': 'value2'}
 
         with pytest.raises(SlackApiError):
-            slack_hook.call(test_method, test_api_params)
```

Comment:
Out of curiosity - was this test working previously?

Reply:
`slack_client_mock.api_call = mock.Mock(side_effect=expected_exception)`

`api_call` method was mocked and mock didn't 100% mimic original signature with only one positional argument. So the test was green before my change.

Reply:
Even more. I had to update test due to **linter** error on CI and not test execution result. The test execution should be green before and after my change.